example_config.yaml

```diff
@@ -1,126 +1,132 @@
 ---
-credentials:
-      # these credentials can be associated
-      # with sources by name in the inventory section
-    - name: root
-      type: 'network'
-      # NOTE: When running quipucords in a container, this path
-      # must be the path that exists in the container.
-      sshkeyfile: ~/.ssh/id_rsa
-      username: root
-    - name: 'vcenter'
-      type: 'vcenter'
-      password: 'example1'
-      username: 'username1'
-    - name: 'sat6'
-      type: 'satellite'
-      password: 'CHANGEME'
-      username: 'CHANGEUSERNAME'
+# General settings. They cover main infrastructure that camayoc interacts
+# with and camayoc itself.
 
-qpc:
-    # this is how you would configure quipucords if running
-    # the development server
-    # if running in a container, set the port to 443
-    # and https to true. Keep ssl-verify false unless you've
-    # set up the certificates
+# Camayoc test framework settings
+camayoc:
+  run_scans: false
```

Comment:
Let's adopt 4 spaces for indentation. If you change this line to 4 spaces, then yamlllint you'll dump these lines to fix:

```
yamllint example_config.yaml
example_config.yaml
  27:3      error    wrong indentation: expected 4 but found 2  (indentation)
  31:3      error    wrong indentation: expected 4 but found 2  (indentation)
  65:81     error    line too long (84 > 80 characters)  (line-length)
  67:3      error    wrong indentation: expected 4 but found 2  (indentation)
  92:3      error    wrong indentation: expected 4 but found 2  (indentation)
  98:7      error    wrong indentation: expected 8 but found 6  (indentation)
  101:7     error    wrong indentation: expected 8 but found 6  (indentation)
```

Reply:
Should I add [yamllint pre-commit](https://yamllint.readthedocs.io/en/stable/integration.html#integration-with-pre-commit) integration while I'm at it?

Reply:
Positively YES.

Reply:
Added in e4c51f8cc801875ca7d7282d571d394fdafeed7a .

I have ran `yamllint` against new config files in discovery-ci MR #1. yamllint does not report any errors, although they are following "2 spaces indentation" pattern. Maybe I would need to also add yamllint configuration file, not sure.

Let me know if you want me to reformat discovery-ci files, or should I leave them as they are. We need both changes merged at roughly the same time, or Jenkins will fail.